pkg/wallet/api.go

```diff
@@ -69,6 +69,10 @@ type Crypto interface {
 	//
 	// error: error
 	DecryptMessage(encMessage []byte, toVerKey string) ([]byte, string, error)
+
+	// FindVerKey picks, out of the keys provided, a key present within the wallet which
+	// the wallet wishes to use for decryption
+	FindVerKey(candidateKeys []string) (int, error)
```

Comment:
Find is not the usual prefix.

Reply:
I was considering `ChooseVerKey` since this chooses a verfication key, how's that?

Reply:
`SelectVerKey`?

Reply:
Is it returning an index? Is that an index of candidateKeys?

Reply:
Find seems better than Choose or Select.

Note: the standard library has a similar function for searching a byte slice. https://golang.org/pkg/bytes/#IndexByte

Reply:
Yeah, it returns an index into `candidateKeys`, indicating the verkey to use for decryption.

Reply:
Leaving it then, thanks!